src/StateService/Network/StateRoot.cs

```diff
@@ -90,7 +90,7 @@ public void SerializeUnsigned(BinaryWriter writer)
 
         public bool Verify(ProtocolSettings settings, DataCache snapshot)
         {
-            return this.VerifyWitnesses(settings, snapshot, 1_00000000);
+            return this.VerifyWitnesses(settings, snapshot, 3_00000000L);
```

Comment:
It's not enough with 2?

Reply:
But it's always `MaxVerificationGas` 🤔.
https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/Helper.cs#L275